app/Services/UserService.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Services;
+use App\Models\User;
+
+class UserService
+{
+    public function store(array $data)
+    {
+        $user = User::create([
+            'name' => $data['name'],
+            'email' => $data['email'],
+            'password' =>$data['password'],
```

Comment:
you can do:

$data['password'] = Hash::make($data['password']);

return User::create($data);